### DISCUSSION TITLE
🔗 Automatically Regenerate CSS & Data After Every Elementor Update (Merged with #28192)

### DISCUSSION THREAD

**Discussion by ghost** [elementor/elementor#28129] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

It is a massive pain (especially for agencies) to visit every Elementor website under management to manually regenerate the files and data. As agencies, we can clear the general cache globally and in an automated fashion, but we still face issues with Elementor's side.

### Describe the solution you'd like.

Automatically Regenerate CSS & Data After Every Elementor Updates

### Describe alternatives you've considered.

Not using Elementor.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by m1ga** [comment]
Having this as an option (default: off) wouldn't hurt but isn't the real issue that you have to run it after an update? I have many pages for years running with "auto-update" for the Elementor plug-in and I never had to go in and regenerate the CSS & data. Using lots of custom css, custom widgets and custom themes.
On reddit there are also people often having issue with this and it's mostly related to caching plugins or server architecture. As a reference: I'm using WP Super Cache and various shared hosting environments (linux based)

**Comment by ghost** [comment]
As far as I know, caching is not the issue, in my case at least. The Elementor updates are ran and the cache gets cleared automatically as part of a directive I give my website management tool.

Immediately after I manually regenerate the E files, the issues go away.

So, I think it would be possible for E to script it out to where things happen in the order they need to, but just so it’s automated.

If they do everything they can on their part (I’m not convinced they have), then we can maybe dive deeper into the overall caching layers and point the “blame” there.

If you Google around, others are reporting this as well.

**Comment by nicholaszein** [comment]
Hi @wpturnedup.

I'm closing this discussion since you've opened another one with more comments #28192.

In the future, please, do not open duplicates. This just increases the time it takes to validate and consider these requests, not to mention it dissolves votes between requests.

Kind regards
